bean-validation/4.0/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Jakarta Validation 4.0 (Under development)"
+date: 2024-03-27
+summary: "Release for Jakarta EE 11"
```

Comment:
This release is for Jakarta EE 12.

Reply:
Good point! 
@starksm64 can you update?